cmd/option.go

```diff
@@ -85,6 +87,15 @@ func (opts *Options) toConfig() *tflint.Config {
 		log.Printf("[WARN] Usage of --only will ignore other rules provided by --enable-rule or --disable-rule")
 	}
 
+	plugins := map[string]*tflint.PluginConfig{}
+	for _, plugin := range opts.EnablePlugins {
+		plugins[plugin] = &tflint.PluginConfig{
+			Name:    plugin,
+			Enabled: true,
+			Body:    nil,
```

Comment:
I'm a little worried about the design of this feature. I find it convenient to be able to enable plugins with the CLI flag, but in that case, the `Body` will always be nil, which will result in incomprehensible errors for plugins that require configuration.

Reply:
One option would be to change the name of the cli option to something like `--enable-plugin-without-config`, then it would be clear that the config is always nil and plugin errors for missing config would be easy to understand. The second option would be to pass the config with the cli option, but I think this is not easy to use or implement.

Reply:
> One option would be to change the name of the cli option to something like --enable-plugin-without-config

Umm... I would like to avoid the descriptive long option name...

> The second option would be to pass the config with the cli option, but I think this is not easy to use or implement.

Agreed.

It's not a problem now, so we have the option to accept this change and consider the design once it's actually a problem. Now I am willing to accept this change.

Reply:
However, since the plugin configuration will be overridden when using the CLI flag, it is necessary to take the same ways as the rules.

https://github.com/terraform-linters/tflint/blob/382e3cc7a5f3d7320c816525caa0769f63bd9943/tflint/config.go#L360-L365

Reply:
I'm not fully sure what you mean. 
I just manually tested my change with  a `.tflint.hcl` config:

```hcl
plugin "codestyle" {
    enabled = false
}

plugin "aws" {
    enabled = true
}
```
The codestyle plugin can be enabled via cli option, this will override the `enabled = false`. Setting the cli option has no effect on the aws plugin block.

Reply:
For instance, if you enable the AWS plugin from the CLI with the following configuration, the value of the `deep_check` option will be overridden.

```hcl
plugin "aws" {
  enabled = false
  deep_check = true
}
```